klp_tc_functions.sh

```diff
@@ -28,7 +28,8 @@ function __klp_add_patched_func() {
 }
 
 function klp_create_header() {
-    sed "s%@@USE_OLD_HRTIMER_API@@%$KLP_TEST_HRTIMER_OLD%" \
+    sed -e "s%@@USE_OLD_HRTIMER_API@@%$KLP_TEST_HRTIMER_OLD%" \
+	-e "s%@@USE_NEW_HRTIMER_SETUP_API@@%$KLP_TEST_USE_NEW_HRTIMER_SETUP_API%" \
```

Comment:
I think it should be changed to something different than OLD and NEW. Versioning or so, with a comment somewhere what is what.

Reply:
sure. Will do. Thanks